rclcpp_action/include/rclcpp_action/client.hpp

```diff
@@ -318,31 +325,47 @@ class Client : public ClientBase
     goal_request->goal = goal;
     this->send_goal_request(
       std::static_pointer_cast<void>(goal_request),
-      [this, goal_request, callback, ignore_result, promise](
+      [this,
+      goal_request,
+      goal_response_callback,
+      feedback_callback,
+      result_callback,
+      ignore_result,
+      promise](
```

Comment:
nitpick: this is a strange indention...

I would have done it like this:

```
[
  this,
  ...,
](...) mutable
{
  // ...
}
```

Reply:
eeed6f5